tests/samplers/test_logprobs.py

```diff
@@ -52,9 +70,18 @@ def test_get_prompt_logprobs(
             "The output text from the top logprob for each token position "
             "should be the same as the output text in the result.")
 
+        # The first prompt logprob is always None
```

Comment:
NOTE: The current vLLM API returns None for the first index prompt logprob. 

So promt_logprob[0] -> the logprob the first token is likely to be generated given the 0th toekn, which doesn't exist. So it is always None.
prompt_logprob[1] -> the logprob the second prompt token is likely to be generated given the first token.